index.html

```diff
@@ -7,5 +7,16 @@
 </head>
 <body>
 
+<div class="container">
+
+    <header class="grid-item grid-a"></header>
+    <main class="grid-item grid-b">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus libero leo, pellentesque ornare, adipiscing vitae, rhoncus commodo, nulla. Fusce quis ipsum. Nulla neque massa, feugiat sed, commodo in, adipiscing ut, est. In fermentum mattis ligula. Nulla ipsum. Vestibulum condimentum condimentum augue. Nunc purus risus, volutpat sagittis, lobortis at, dignissim sed, sapien. Fusce porttitor iaculis ante. Curabitur eu arcu. Morbi quam purus, tempor eget, ullamcorper feugiat, commodo ullamcorper, neque.</main>
+    <aside class="grid-item grid-c"></aside>
+    <aside class="grid-item grid-d"></aside>
+    <aside class="grid-item grid-e"></aside>
+    <footer class="grid-item grid-f"></footer>
+
```

Comment:
Door een simpele content in je html te gebruiken in je aside-classes kan je vlakken wat overzichtelijker maken en je padding ook echt gebruiken.  

Je had er bijvoorbeeld <header class="grid-item grid-a">A</header> van kunnen maken zodat de vlakken iets minder gecropt zijn.